cli/core/go.mod

```diff
@@ -38,6 +39,7 @@ require (
 	github.com/vmware-tanzu/tanzu-framework/apis/cli v0.0.0-00010101000000-000000000000
 	github.com/vmware-tanzu/tanzu-framework/capabilities/client v0.0.0-00010101000000-000000000000
 	github.com/vmware-tanzu/tanzu-framework/cli/runtime v0.0.0-00010101000000-000000000000
+	github.com/vmware-tanzu/tanzu-framework/pinniped-components/common v0.0.0-00010101000000-000000000000
```

Comment:
Fake `version v0.0.0-00010101000000-000000000000` in our `require` so the `replace` can use the local.  We do intend to pick up the extraction of the whole `/pinniped-components` directory at some point. Not sure what the timeline is for eviction from TF at this point.

Reply:
I'll tag @prkalle since we have been discussing this topic as of late.

Reply:
I believe @jmoroski is overseeing the whole TF refactor and eviction effort, perhaps also interested in what we are doing here.  (see description & one of my other comments for summary).

Reply:
If other code (`cli plugins` or `cli core commands`, etc) move out of TF before `/pinniped-components` does, but then references our `<TF>/pinniped-components` I wonder if we have a risk of circular dependency.